README.rst

```diff
@@ -207,6 +207,13 @@ Run tests with:
 
     ./runtests.py
 
+Visualizer
+----------
```

Comment:
Remove this heading and then move the image to the top, after the following line of text:

`OpenWISP 2 network topology module (built using Python and Django web-framework).`

Reply:
after or before the table of content?

Reply:
owh i get it